Libraries/LibWeb/CSS/CSSStyleDeclaration.cpp

```diff
@@ -480,8 +514,8 @@ JS::ThrowCompletionOr<bool> CSSStyleDeclaration::internal_set(JS::PropertyKey co
         return Base::internal_set(name, value, receiver, cacheable_metadata);
 
     auto css_text = value.is_null() ? String {} : TRY(value.to_string(vm));
-
-    TRY(Bindings::throw_dom_exception_if_needed(vm, [&] { return set_property(property_id, css_text); }));
+    auto name_str = StringView { name.to_string().characters(), name.to_string().length() };
```

Comment:
`name.to_string().bytes_as_string_view()` is the easier way to do this.

Reply:
Looks like this says bytes_as_string_view does not exist.
```
/Users/ankush/codespace/ladybird/Libraries/LibWeb/CSS/CSSStyleDeclaration.cpp:517:38: error: no member named 'bytes_as_string_view' in 'AK::ByteString'
  517 |     auto name_str = name.to_string().bytes_as_string_view();
      |                     ~~~~~~~~~~~~~~~~ ^
/Users/ankush/codespace/ladybird/Libraries/LibWeb/CSS/CSSStyleDeclaration.cpp:518:9: error: no matching function for call to 'throw_dom_exception_if_needed'
  518 |     TRY(Bindings::throw_dom_exception_if_needed(vm, [&] { return set_property(property_id, name_str, css_text); }));
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I will be updating the PR along the rest of the comments. If this needs change, will update that on your comment.

Reply:
Oh right, I thought it was a `String` but it's a `ByteString`. So it's `name.to_string().view()`.